File: front_end/lib/features/meeting/domain/entities/meeting.dart
class Meeting {
  final String meetingId;
  final String title;
  final String createdBy;
  final String description;
  final String location;
  final DateTime date;
  final String startTime;
  final String endTime;
  final DateTime createdAt;
  final DateTime? updatedAt;
  final bool isOwner;

  const Meeting({
    required this.meetingId,
    required this.title,
    required this.createdBy,
    required this.description,
    required this.location,
    required this.date,
    required this.startTime,
    required this.endTime,
    required this.createdAt,
    this.updatedAt,
    required this.isOwner,
  });
}

File: back-end/meeting_service/src/main/java/com/kynn/meeting_service/entity/Meeting.java
package com.kynn.meeting_service.entity;

import jakarta.persistence.*;
import lombok.Data;
import org.hibernate.annotations.CreationTimestamp;

import java.time.Instant;
import java.time.LocalDate;
import java.time.LocalTime;
import java.util.Date;
import java.util.UUID;

@Entity
@Data
@Table(name = "meeting")
public class Meeting {

  @Id
  @GeneratedValue(strategy = GenerationType.UUID)
  private UUID meetingId;

  private UUID createdBy;

  private String title;
  private String description;
  private String location;
  private LocalDate date;
  private LocalTime startTime;
  private LocalTime endTime;


  @CreationTimestamp
  private Instant createdAt;
  private Instant updatedAt;

}
